examples/units_and_coordinates/PSP_trag_sunpy.py

```diff
@@ -0,0 +1,50 @@
+"""
+============================================
+Getting the trajectory of Parker Solar Probe
+============================================
+
+This example shows how to plot the trajectory of Parker Solar Probe within the solar system using
+`astropy's solar system ephemeris <http://docs.astropy.org/en/stable/coordinates/solarsystem.html#solar-system-ephemerides>`__ information and sunpy.
+"""
+import datetime
+
+import matplotlib.pyplot as plt
+
+from astropy.time import Time
+
+from sunpy.coordinates import get_body_heliographic_stonyhurst
+from sunpy.coordinates.ephemeris import get_horizons_coord
+
+##############################################################################
+# You can make use of `~datetime.datetime.now` to pass current date and time.
+# Now can use `get_horizons_coord() <https://docs.sunpy.org/en/stable/generated/api/sunpy.coordinates.get_horizons_coord.html>`__
+# to get the coordinates of Parker Solar Probe.
+
+today = datetime.datetime.now()
+obstime = Time(today)
```

Comment:
As I did in the preview example, you should be using parse_time from sunpy and passing in "now".

Reply:
Copy that